### PR TITLE
#5 :memo: Tranche 2 of documentation migration

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**
Tranche 2
* [Versions](https://boozallen.github.io/aissemble/aissemble/current/aissemble-versions.html)
* [aiSSEMBLE Baseline Process](https://boozallen.github.io/aissemble/aissemble/current/solution-baseline-process.html)
* [aiSSEMBLE Approach](https://boozallen.github.io/aissemble/aissemble/current/aissemble-approach.html)
* [Add Pipelines](https://boozallen.github.io/aissemble/aissemble/current/add-pipelines-to-build.html)
* [Pipeline Metamodel](https://boozallen.github.io/aissemble/aissemble/current/pipeline-metamodel.html)

### CODE REVIEW

File: docs/modules/ROOT/pages/pipeline-metamodel.adoc [hunk 1]
```diff
@@ -0,0 +1,909 @@
...
+| The name of the configuration key-value pair by which the value can be retrieved.
+
+| `steps/step/configuration/value`
+| No
+| None
+| The configuration value.
+
+|===
+
+== Pipeline Model Use Case
```
Comment (nartieri) [inline docs/modules/ROOT/pages/pipeline-metamodel.adoc:902]: C: As discussed, recommend removing so as not to expose publicly.

**Review by nartieri — COMMENTED** [review]
One Critical comment. The rest looked good to me.

File: docs/modules/ROOT/images/aissemble-solution-architecture.png
Comment (nartieri) [inline docs/modules/ROOT/images/aissemble-solution-architecture.png:1]: C: Per conversation, suggest redacting Reusable Pipeline names to not expose them publicly.

**Review by nartieri — COMMENTED** [review]
Added a second critical item based on the image updates we discussed.

File: docs/modules/ROOT/images/aissemble-solution-architecture.png
Comment (d-ryan-ashcraft) [reply docs/modules/ROOT/images/aissemble-solution-architecture.png:1]: ✅ RESOLVED

**Pull request merged**